README/ReleaseNotes/v630/index.md

```diff
@@ -20,6 +20,7 @@ The following people have contributed to this new version:
  Will Buttinger, Rutherford Appleton Lab,\
  Philippe Canal, FNAL,\
  Olivier Couet, CERN/EP-SFT,\
+ Marta Czurylo, CERN/EP-SFT,\
```

Comment:
YIKES sorry! Can you check that I didn't forget you in v6-30-00-patches (which is where we build the release notes from, i.e. the branch that actually matters for this change)? Please also backport this to v6-30-00-patches!